### ISSUE TITLE
VWI manchester encoding help

### ISSUE THREAD

**Issue by HansSanitizer** [mikaelpatel/Cosa#178]
Hi, I've recently been playing around with using cosa for a wireless school project. I got the sending and recieving down, but I am curious as to what is actually sent using the manchester codec. So far I have determined that the preample is sent without encoding to the best of my knowledge, with the default settings it looks like 8 bytes of data is sent as the preample ending with 0x5d. What I'm wondering is if the 0x555d specified in the codec properties makes up the 7th and 8th byte of data being sent as it appears to be. I am also curious as to what data is being sent after that. After the preample I assume that the destination is sent being 2 bytes of data, not sure if it's manchester encoded or not, then 2 bytes for the port, then the buffer which I would assume is manchester encoded, then some sort of checksum.

TL;DR?
I'm looking for a bit more of a descriptive packet overview for using the send function in the cosa library.
